vision_msgs_meta/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(vision_msgs_meta NONE)
```

Comment:
No meta package is required, lets keep them separated. You can make the rviz plugins though tied to the msgs (as would be needed) so they'd come in pairs if you grabbed the rviz plugins.